File: Trabalho1/Torre.py
class Torre:
    def __init__(self, id, nome, endereco) -> None:
        self.id = id
        self.nome = nome
        self.endereco = endereco

    def getNome(self):
        return self.nome

    def __str__(self):
        return "ID da torre: "+ str(self.id)+" - Nome da torre: "+ self.nome+ " - Endereço: "+ self.endereco

File: Trabalho1_final/Torre.java
package trabalho.Trabalho1_Java;

public class Torre {
    public int id;
    public String nome;
    public String endereco;

    public Torre(int id, String nome, String endereco){
        this.id = id;
        this.nome = nome;
        this.endereco = endereco;
    }

    public int getId() {
        return id;
    }

    public String getNome() {
        return nome;
    }

    public String toString(){
        return "ID. Torre: " + id + ", Nome: " + nome + ", Endereco: " + endereco;
    }
}
